### ISSUE TITLE
Uncaught ReferenceError: dnnpagesSelectBox is not defined

### ISSUE THREAD

**Issue by damirt2017** [w8tcha/dnnckeditor#26]
I am not sure if this is the right project to report this issue to. We got this error when we were testing image uploads. There was already an image in the editor and we uploaded a second image. 

To reproduce this error, paste this code into a  ckeditor, save it, close the editor and load it again. You will see some 404 errors from missing images so please ignore those.

```
<p>
<span contenteditable="false" tabindex="-1">
<a data-widget="image" href="/">
<img alt="" src="/portals/pitbullpowersportsinc/banner.jpg" style="width: 100%;" />
</a>

<span style="background: url(&quot;http://pitbullpowersportsinc.edevpod1-dnnbuild1.dx1app.com/Providers/HtmlEditorProviders/CKEditor/plugins/widget/images/handle.png&quot;) rgba(220, 220, 220, 0.5); top: -15px; left: 0px; display: block;">
<img draggable="true" height="15" role="presentation" src="data:image/gif;base64,R0lGODlhAQABAPABAP///wAAACH5BAEKAAAALAAAAAABAAEAAAICRAEAOw==" title="Click and drag to move" width="15" /></span></span>

</p> 
```

and in the developer console you will get this error:

Uncaught ReferenceError: dnnpagesSelectBox is not defined
    at parseLinkAttributes (plugin.js?t=HBDD:504)
    at g.init (plugin.js?t=HBDD:413)
    at new g (ckeditor.js:761)
    at p.initOn (ckeditor.js:805)
    at p.initOnAll (ckeditor.js:806)
    at a.<anonymous> (ckeditor.js:783)
    at a.m (ckeditor.js:10)
    at a.<anonymous> (ckeditor.js:12)
    at a.CKEDITOR.editor.CKEDITOR.editor.fire (ckeditor.js:13)
    at ckeditor.js:1315

**Comment by w8tcha** [comment]
At the moment i can not reproduce the issue with the latest version.  Which version did you use?

Can you describe the original steps to reproduce the issue?

> Uncaught ReferenceError: dnnpagesSelectBox is not defined

Normally the error would only show up if the Tabs.ashx is not loaded correctly

**Comment by damirt2017** [comment]
Thanks for looking into it. It happens when you drag and drop another image to a content that already has an image. If I find a better way to reproduce this I will open a new issue. Thanks again.

**Comment by w8tcha** [comment]
Issue has been fixed in 02.01.13

**Comment by damirt2017** [comment]
Thanks a lot. We will try it on our system next week.
